modules/cooler/zoomify/functions.nf

```diff
@@ -0,0 +1,68 @@
+//
+//  Utility functions used in nf-core DSL2 module files
+//
+
+//
```

Comment:
Need to update this file?

Reply:
linting is passing now, so I guess this should be fine, right?

Reply:
yeah, that's the latest.